test/functional/cypress/specs/pipeline/my-pipeline-edit.js

```diff
@@ -4,11 +4,14 @@ const {
   assertFieldRadios,
   assertBreadcrumbs,
   assertFieldInput,
+  assertFieldTypeahead,
 } = require('../../support/assertions')
 
 const selectors = require('../../../../selectors')
 
-const pipelineItem = pipelineItemLambdaPlc.results[0]
+const formSelectors = selectors.pipelineForm
+const firstItem = pipelineItemLambdaPlc.results[0]
+const secondItem = pipelineItemLambdaPlc.results[1]
```

Comment:
Optional: These could be defined in a single line.
```suggestion
const [firstItem, secondItem] = pipelineItemLambdaPlc.results
```